extraConfigDpuTenant.py

```diff
@@ -141,197 +154,125 @@ def ExtraConfigDpuTenant(cc: ClustersConfig, cfg: ExtraConfigArgs, futures: Dict
+        for bfmap in cfg.mapping:
+            a: Dict[str, str] = {}
+            mp = re.sub(r'np\d$', '', bf_port)
+            a["OVNKUBE_NODE_MGMT_PORT_NETDEV"] = f"{mp}v0"
+            contents += f"  {bfmap['worker']}: |\n"
+            for k, v in a.items():
+                contents += f"    {k}={v}\n"
+        open("/tmp/1.yaml", "w").write(contents)
+
+        logger.info("Running create")
+        logger.info(tclient.oc("create -f /tmp/1.yaml"))
 
-    logger.info("Running create")
-    logger.info(tclient.oc("create -f /tmp/1.yaml"))
-
-    for nc in cc.workers:
-        cmd = f"label node {nc.name} network.operator.openshift.io/dpu-host="
+    for e in cc.workers:
+        cmd = f"label node {e.name} network.operator.openshift.io/dpu-host="
         logger.info(tclient.oc(cmd))
-        ip = tclient.get_ip(nc.name)
+        ip = tclient.get_ip(e.name)
         if ip is None:
-            logger.error(f"Failed to get ip for node {nc.name}")
+            logger.error(f"Failed to get ip for node {e.name}")
             sys.exit(-1)
         rh = host.RemoteHost(ip)
         rh.ssh_connect("core")
-        # workaround for https://issues.redhat.com/browse/NHE-335
+
+        if new_api:
+            # DELTA: "ovn-k8s-mp0" port is deleted by OVN-K. https://github.com/ovn-org/ovn-kubernetes/pull/3571
+            result = rh.run("sudo ovs-vsctl show")
+            if "ovn-k8s-mp0" in result.out:
+                logger.info(result.out)
+                logger.info("Unexpected ovn-k8s-mp0 interface found in br-int.")
+                # FIXME: The above patch did not seem to entirely work. We will need to investigate further
+                # For now we will delete the port.
+        else:
+            # workaround for https://issues.redhat.com/browse/NHE-335
+            pass
+
         logger.info(rh.run("sudo ovs-vsctl del-port br-int ovn-k8s-mp0"))
 
     logger.info("creating mc to disable ovs")
-    # At this point we error out, because the patch ports on the DPU OvS side does not get created
-    # patch ports are created on the OVN side. TODO: FIXME For now try this to make OVN-K on DPU happy:
-    for e in cc.workers:
-        cmd = f"annotate --overwrite node/{e.name} k8s.ovn.org/node-mgmt-port={{\"PfId\":0,\"FuncId\":0}}"
-        logger.info(cmd)
-        logger.info(tclient.oc(cmd))
+    if not new_api:
+        # At this point we error out, because the patch ports on the DPU OvS side does not get created
+        # patch ports are created on the OVN side. TODO: FIXME For now try this to make OVN-K on DPU happy:
+        for e in cc.workers:
+            cmd = f"annotate --overwrite node/{e.name} k8s.ovn.org/node-mgmt-port={{\"PfId\":0,\"FuncId\":0}}"
+            logger.info(cmd)
+            logger.info(tclient.oc(cmd))
 
     tclient.oc("create -f manifests/tenant/disable-ovs.yaml")
     logger.info("Waiting for mcp")
-    tclient.wait_for_mcp("dpu-host", "dpu host mode")
+    if new_api:
```

Comment:
Same deal here, this is just a difference in the string that is logged, disable-ovs.yaml is fine for both